labs/lab0/chapter0.md

```diff
@@ -1,33 +1,30 @@
 ## Introduction
 
 In order to make this lab simple to work with, we are going to leverage
-a tool known as the Container Development Kit (CDK). The CDK is a prebuilt 
-Vagrant VM with RHEL installed. The VM also has all the tools you need for
-building and developing software with containers. The included
-software provides docker, kubernetes and, optionally, OpenShift. 
+a product known as the Container Development Kit (CDK). The CDK leverages a tool called [minishift](https://github.com/minishift/minishift) to setup a RHEL VM with docker and OpenShift already installed. Optionally, you can register with your Red Hat Account and actually get updates to the VM as well. The CDK also includes documentation, getting started and howto guides, and a community of users to learn from.
```

Comment:
I wouldn't mention getting updates on the VM. I'm not sure we want people updating minishift "in place".